RELEASE-NOTES.txt

```diff
@@ -1,3 +1,10 @@
+1.0.7
+=====
+* Update instructions of translation process #381
+* Remove test timers related to code block #380
+* Avoid data corruption due to writing concurrently in the user data file #366
+* Allow blocking user from adding demo sites #354
```

Comment:
```suggestion
* Display a meaningful message for users who has demo sites blocked #354
```

Reply:
Thanks for the suggestions 🙇 ! I've updated this entry in https://github.com/Automattic/studio/pull/412/commits/be5c825387362755911d8fb8897b32a8decc8165.